kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -286,10 +259,12 @@ private Subscription createSubscriptionViaMcts(Subscriber subscriber, Subscripti
             }
         } else if (subscriber.getLastMenstrualPeriod() != null && subscriber.getDateOfBirth() == null &&
```

Comment:
Yeah, it's really gross logic described in #159. I'll add the text of that spec section as a javadoc comment for this method though.